client/pages/fetch-data.jsx

```diff
@@ -71,14 +146,50 @@ export default class FetchData extends React.Component {
   }
 
   render() {
+    const value = this.state.wagerAmount;
     if (!this.state.fixturesList.length) {
       <NoGamesToday/>;
     }
     return (
       this.state.isLoading
         ? <p>isLoading...</p>
         : <>
-        <FixturesList toggleMatchDetails= {this.state.toggleMatchDetails} activeId = {this.state.activeId} fixtures ={this.state.fixturesList} click={id => this.handleClick(id)} teamDetails = {this.state.teamDetails}loading={this.state.isLoading} clickTeamDetails={() => this.handleTeamDetails()}/>
+        <FixturesList wagerAmount={this.state.wagerAmount} homeOdds={this.state.homeOdds} awayOdds = {this.state.awayOdds} betOn = {this.state.betOn} toggleMatchDetails= {this.state.toggleMatchDetails} activeId = {this.state.activeId} fixtures ={this.state.fixturesList} click={id => this.handleClick(id)} teamDetails = {this.state.teamDetails}loading={this.state.isLoading} betTeamId ={this.betTeamId}
+        addWagerTeam = {this.addWagerTeam}
+        />
+  <>
+  <div className = {!this.state.betTeamId ? 'hidden' : ''}>
+
+  <div className="row column-full center" >
+         <div className="outer-card column-full">
+           <div className="match-card row center">
+
+         <img className={'team-logo'} src={this.state.teamLogo} alt=''/>
+      <div className = "input-container column-full">
+        <form onSubmit={this.handleSubmit} className="column-full" >
+          <input
+        className = "wager-input"
+        type="number"
+        max = {this.state.userTokens}
+        required
+        autoFocus
+        value = {value}
+        placeholder= "WAGER HERE"
+        onChange={this.handleChange}/>
+        <div>
+          <h4 className={'row payout center'}>{this.state.script}</h4>
+          </div>
+        <div className = "button-container row">
+      <button className = "enter-button " type="submit" >Enter</button>
+      <button onClick = {this.checkProfit} className = "enter-button" type="button" >Check Profit</button>
+        </div>
+        </form>
+      </div>
+        </div>
+    </div>
+        </div>
+</div>
+      </>
```

Comment:
Please take some time to fix the indentation here, it's pretty tough to see what is going on